cmd/xcode.go

```diff
@@ -62,9 +70,13 @@ func initExportOutputDir() (string, error) {
 }
 
 func scanXcodeProject(cmd *cobra.Command, args []string) error {
-	absExportOutputDirPath, err := initExportOutputDir()
-	if err != nil {
-		return fmt.Errorf("failed to prepare Export directory: %s", err)
+	absExportOutputDirPath := ""
```

Comment:
NOTE: var absExportOutputDirPath string, would explicitly show that we do not care / use the default value of the var